packages/@atjson/conventional-commits/README.md

```diff
@@ -0,0 +1,86 @@
+# ✌️ conventional-commits
+
+[Conventional Commits](https://conventionalcommits.org/) is a community standard designed to make communicating software changes and for these changes to be generated by a software without someone having to write these changes by hand. Our software that we use to manage `atjson`, [`lerna`](https://github.com/lerna/lerna/tree/master/commands/publish#--conventional-commits), supports this standard.
```

Comment:
"... to make communicating software change" easier to understand?

"to be generated by ~a~ software"

Reply:
This is kludgy and I will reword 😅